src/utils/spinel_encoder.hpp

```diff
@@ -33,18 +33,15 @@
 #ifndef SPINEL_ENCODER_HPP_
 #define SPINEL_ENCODER_HPP_
 
-#include "openthread-core-config.h"
-
 #include <openthread/ip6.h>
 #include <openthread/message.h>
 #include <openthread/ncp.h>
 
-#include "openthread-core-config.h"
-#include "ncp/ncp_buffer.hpp"
-#include "ncp/spinel.h"
+#include "spinel.h"
+#include "spinel_buffer.hpp"
 
 namespace ot {
-namespace Ncp {
+namespace Utils {
```

Comment:
How about using `Spinel` instead of `Utils` as namespace name for all the spinel modules?
This way we distinguish it from `ot:Utils` namespace from `src/core/utils` modules.